### ISSUE TITLE
Exporting Logits

### ISSUE THREAD

**Issue by sahar90o** [Kamnitsask/deepmedic#223]
Hello, 

I want to know how it is possible to export the predicted logits during validation and test time. I know that DeepMedic automatically exports probability maps during test time, but I need the predictions one step before applying Softmax. I would appreciate it if someone could help. 

Bests, 
Zahra

**Comment by Kamnitsask** [comment]
Hello and sorry for the late reply -- really busy term between teaching etc, apologies...

If you are still looking into this, you may find some useful info in these previous issues:

https://github.com/deepmedic/deepmedic/issues/129
https://github.com/deepmedic/deepmedic/issues/123

These ask about how to visualise features, and essentially the logits are the "feature maps" of the last layer.

So, I believe you could save them by commenting out the below entry in the testConfig (e.g. at deepMedic/test/testConfig.cfg):
(for an example when you have 4 classes)
minMaxIndicesOfFmsToSaveFromEachLayerOfFullyConnectedPathway = [[],[],[0,4]]

I hope this helps!
